### ISSUE TITLE
Request to join this team

### ISSUE THREAD

**Issue by alott31** [BYUCS452F2021/sport-events#5]
Hi! I really liked the idea that you presented in class. I am always looking for people to play volleyball with so I understand the need for an app like this.  I am not as comfortable with web development, but I have a lot of experience working with large databases.  This summer I worked as a Data Engineering intern at Texas Instruments. I would love to help with the database portion of this project and I would be willing to help with other portions of the project as needed. Thank you for proposing this project idea and for considering adding me to your team.

**Comment by victorvel** [comment]
Yes, definitely! Welcome to team. What is your name? I'll add you to our slack message so we can all meet sometime this week. Thanks!

**Comment by alott31** [comment]
My name is Ashly Lott
